crowbar_framework/app/models/proposal.rb

```diff
@@ -137,7 +137,7 @@ def status
     else
       return "unready" if bc["crowbar-committing"]
       return "pending" if bc["crowbar-queued"]
-      return "hold" if !bc.has_key? "crowbar-queued" and !bc.has_key? "crowbar-committing"
+      return "hold" if !bc.key? "crowbar-queued" and !bc.has_key? "crowbar-committing"
```

Comment:
Style/AndOr: Use `&&` instead of `and`. (https://github.com/bbatsov/ruby-style-guide#no-and-or-or)<br>Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`. (https://github.com/bbatsov/ruby-style-guide#hash-key)